.github/workflows/ci.yaml

```diff
@@ -117,6 +117,8 @@ jobs:
     steps:
       - name: Fetch sources
         uses: actions/checkout@v4
+        with:
+          submodules: true
```

Comment:
You want to update [`cookiecutter/{{cookiecutter.github_repo_name}}/.github/workflows/ci.yaml`](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/cookiecutter/%7B%7Bcookiecutter.github_repo_name%7D%7D/.github/workflows/ci.yaml) instead (this project doesn't have submodules.

And you probably need to do that on all jobs where some form of `pip install -e .` is used.

For which you'll also need to update the *golden tests* as [described in the contributing guide](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/CONTRIBUTING.md#golden-tests).